Assets/Scripts/Main.cs

```diff
@@ -4,41 +4,19 @@
 
 public class Main : MonoBehaviour
 {
-    public GameObject playerHand;
     public GameObject playSlot;
-
-    public GameObject simpleCardPrefab;
-    public GameObject[] facesPrefabs;
+    public GameObject playerHand;
```

Comment:
You can refer HandDrower component directly, since you know you need it in the end
```suggestion
    public HandDrower playerHand;
```

Reply:
Well, nice to know that but I did it for following reason actually: `playerHand` is the scene object that could be used further in code and it's not obvious then. For example, if I want to calculate `playerHand` position on the scene and set it to the middle of the scene I need to use GameObject since this object has a coordinates